src/SqlBindingExtension.cs

```diff
@@ -28,5 +29,22 @@ public static IWebJobsBuilder AddSql(this IWebJobsBuilder builder, Action<SqlOpt
 
             return builder;
         }
+
+        internal static IWebJobsBuilder AddSqlScaleForTrigger(this IWebJobsBuilder builder, TriggerMetadata triggerMetadata)
+        {
+            IServiceProvider serviceProvider = null;
+            var scalerProvider = new Lazy<SqlScalerProvider>(() => new SqlScalerProvider(serviceProvider, triggerMetadata));
+            builder.Services.AddSingleton((Func<IServiceProvider, IScaleMonitorProvider>)delegate (IServiceProvider resolvedServiceProvider)
+            {
```

Comment:
is incremental scaling already used in public preview? if not is it possible to block use of incremental scaling  - @alrod ?

Reply:
Yes, it looks like we can ger rid of incremental scaling(IScaleMonitorProvider) here as we have never released it.

Reply:
It is in the present in the public docs https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql-trigger?tabs=isolated-process%2Cportal&pivots=programming-language-csharp#enable-runtime-driven-scaling